backend/src/main/java/com/bakdata/conquery/resources/api/ResultCsvResource.java

```diff
@@ -30,7 +31,7 @@
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
-@Path("result/")
+@Path("result/csv")
```

Comment:
Api Änderung bitte in openapi eintragen, und ist das JupyEnd schon informiert über die Änderung?

Reply:
Das muss ich noch klären, aber an sich bekommt es wie das Frontend die URL als dem ExecutionStatus und leitet sich die nicht selber her.

@MaxVonKnobloch ist das ein Problem aus Sicht der CqAPI wenn der Pfad sich hier ändert? Wie matched du momentan auf das Arrow-Format

Reply:
@awildturtok ist noch nicht in OpenApi drinne ;)